scripts/rules/types/auth0-rules.d.ts

```diff
@@ -2,14 +2,7 @@
 export {}
 
 declare global {
-  namespace NodeJS {
-    interface Global {
-      CACHE_KEY: string
```

Comment:
Seems fine to get rid of this because we currently don't use any cached info in our rules, but noting that [this is a suggested key based on the `auth0-rules-typings` package](https://github.com/tepez/auth0-rules-types#add-typings-for-cache).